index.js

```diff
@@ -1 +1,33 @@
-console.log('welcome to thirty days of javascript');
\ No newline at end of file
+console.log('welcome to thirty days of javascript');
+
+/*string data type*/
+let myString ="javascript";
```

Comment:
![image](https://github.com/user-attachments/assets/865d887b-157d-47a8-adf0-fde5592b1115)
You should use `console.log(myString)` to get `javascript` in the output.